src/fal/cli/flow_runner.py

```diff
@@ -14,12 +14,16 @@ def fal_flow_run(parsed):
     node_graph = NodeGraph.from_fal_dbt(fal_dbt)
     execution_plan = ExecutionPlan.create_plan_from_graph(parsed, node_graph)
 
-    run_scripts(_id_to_fal_scripts(node_graph, execution_plan.before_scripts), project)
+    before_scripts = _id_to_fal_scripts(node_graph, execution_plan.before_scripts)
+    results = run_scripts(before_scripts, project)
+    handle_run_results(before_scripts, results)
```

Comment:
let's call this something like `throw_if_failed`

otherwise I am good with this as discussed on Tuesday